plugins/Satellites/src/Satellite.hpp

```diff
@@ -117,13 +125,13 @@ class Satellite : public StelObject
 	enum OptStatus
 	{
 		StatusOperational		= 1,
```

Comment:
I think it'd be prettier to start this list with StatusUnknown=0 and else remove the explicit numbers.

Reply:
here is another point of view: by putting a zero/default value at the bottom, whenever someone inserts new enums, there will never be a need to juggle with comma's at the end of the list.

(In Python, a list can end with a comma, not in C++).

Reply:
I don't understand the difficulty of "juggling with commas". Is there a difficulty with formatting a list of enums?  These are simple consecutive enums, the numerical value of which don't have to be given explicitly. 

TBH the plugin is not my code and I may have overlooked something.